### ISSUE TITLE
DSM 7 Keine Verbindung

### ISSUE THREAD

**Issue by Trrw** [iobroker-community-adapters/ioBroker.synology#102]
Ich bekomme mit DSM7 leider keine Verbindung. Aktuell verwendet ich die Version 2.0.1.
Meine Synology ist per 2FA gesichert. Dies habe ich testweise auch deaktiviert um zu schauen ob es daran liegt. Geblockt ist nicht die IP zum Anmelden. 

Beim verbinden bekomme ich folgende Fehlermeldung
<img width="1221" alt="Bildschirmfoto 2022-01-22 um 10 04 17" src="https://user-images.githubusercontent.com/24519230/150632215-4a224117-bb61-4196-a077-fb14c988c8f8.png">
:

Nach einem Neustart vom ioBroker:
<img width="1314" alt="Bildschirmfoto 2022-01-22 um 10 21 05" src="https://user-images.githubusercontent.com/24519230/150632814-c61a74d1-a6a9-49d9-bfb9-16991ab37ad2.png">

**Comment by CBRocker** [comment]
Ich habe neuerdings das gleiche Problem.
Nach Installation DSM 7.0.1 und Adapter-Version 2.0.1 (Git) hat der Zugriff auf die Synology wieder funktioniert.
Aber seit kurzem wieder nicht mehr. Ich habe kürzlich die DSM-Version 7.0.1 Update 2 installiert. Kann es damit zusammenhängen?
Und bitte das Kennwort-Feld wieder encrypten :-)
Danke, ist ein klasse Adapter.

**Comment by Apollon77** [comment]
Please provide a debug level log

**Comment by arteck** [comment]
check new version

**Comment by MC-1984** [comment]
I have just installed 2.1.5 and the Adapter does not connect any more. 

Log level even set on silly does not provide more than that:

`
synology.1 | 2022-04-03 19:06:24.485 | info | Connecting to Synology 192.168.15.101:5101
-- | -- | -- | --
synology.1 | 2022-04-03 19:06:24.434 | info | starting. Version 2.1.5 (non-npm: iobroker-community-adapters/ioBroker.synology#177ffb4071b0d45f6fa436f487b519a896c54770) in /opt/iobroker/node_modules/iobroker.synology, node: v14.19.1, js-controller: 4.0.21
synology.1 | 2022-04-03 19:06:17.965 | info | Terminated (ADAPTER_REQUESTED_TERMINATION): Without reason

`

Worked perfectly with 2.0.1 before (besides reboot & shutdown). I reentered the password. Any ideas what the problem is?

**Comment by Apollon77** [comment]
Are you sure that you really set the loglevel of the instance? Admin - Instances  enable expertmode - expand instance - change loglevel ... Can you try older versions inbetween to maybe see where it broke?

**Comment by MC-1984** [comment]
@Apollon77 Not sure when it started but there are now other messages in the log. I probably was too impatient and did not wait long enough. Although it has been at leas 2 minutes that I waited.

The messages in the log suggest that I did not enter a 2-step verification code. But a 2FA code is entered. I even reseted it and entered the new 2FA code as well as the password. No change. 

How can I install older versions of the adapter from Github to find the last working version?

`
synology.1 | 2022-04-03 23:59:35.314 | debug | *** ERROR : src: *sendPolling syno[dsm][listPackages] code: 403 message: 2-step verification code required
-- | -- | -- | --
synology.1 | 2022-04-03 23:59:35.313 | debug | * No response, read next.
synology.1 | 2022-04-03 23:59:34.909 | debug | * Get info from (firstPoll) api: DSM method: listPackages params: {"version":1}
synology.1 | 2022-04-03 23:59:34.908 | debug | * sendPolling. namePolling = firstPoll \| iteration = 0 \| typeof poll = object \| poll = {"api":"dsm","method":"listPackages","params":{"version":1}}
synology.1 | 2022-04-03 23:59:34.907 | debug | -----------------------------------------------------------------------------------------------------
`
